jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -62,9 +62,9 @@ YUI.add('user-profile', function() {
     _fetchMacaroonCallback: function(error, macaroon) {
       if (error) {
         console.log(error);
-      } else {
-        this.props.storeUser('charmstore', true);
+        return;
       }
+      this.props.storeUser('charmstore', true);
```

Comment:
This isn't a problem with the code, but I'm curious why you prefer this structure to the if/else that was there before.